internal/wasm/sdk/test/e2e/exceptions.yaml

```diff
@@ -1,3 +1,4 @@
 # Exception Format is <test name>: <reason>
-"jsonpatch/set": "unexpected panic or evaluation error - https://github.com/open-policy-agent/opa/issues/2949"
-"jsonpatch/json_patch_tests": "unexpected panic or evaluation error - https://github.com/open-policy-agent/opa/issues/2949"
+"eqexpr/unordered: object keys": "https://github.com/open-policy-agent/opa/issues/3351"
+"eqexpr/unordered: object keys (reverse)": "https://github.com/open-policy-agent/opa/issues/3351"
+"regexfindallstringsubmatch/multiple patterns, matches, and captures with specified number of matches": "https://github.com/open-policy-agent/opa/issues/3352"
```

Comment:
Out with the old, in with the new..... gist of this: the `sort_bindings` implementation in the wasm e2e test runner was broken, so anything that had set `sort_bindings: true` would automagically **pass**, regardless of the results. These have been uncovered that way -- we're lucky, it's not that much 😄